firmware_DavidBoard/Core/Src/Controller.c

```diff
@@ -130,18 +136,18 @@ static Controller_State_t DoMath_State(void)
 {
     static int loopCounter;
     static int comp_off_counter;
-    //get 5 temp readings
-    float temps[5] = {DataBuffer.temperature[0], DataBuffer.temperature[1], DataBuffer.temperature[2], DataBuffer.temperature[3], DataBuffer.temperature[4]}; //5 temp probe readings
-    float temp_diffs[5]; //to store deltas between probe readings and avg temp
+    //get 4 temp readings
+    float temps[4] = {DataBuffer.temperature[0], DataBuffer.temperature[1], DataBuffer.temperature[2], DataBuffer.temperature[3]}; //5 temp probe readings
```

Comment:
I said I was going to become the owner of this section.  Regardless, it's alright to keep when we merge your stuff in, My stuff will overwrite yours.